.github/workflows/test.yml

```diff
@@ -4,35 +4,11 @@ on: [push]
 
 jobs:
   xcode_tests:
-    name: ${{ matrix.platform }} Tests (Xcode ${{ matrix.xcode }})
+    name: Tests
     runs-on: macos-latest
-    strategy:
-      fail-fast: false
-      matrix:
-        xcode: ["11.5"]
-        platform: ["iOS"]
 
     steps:
       - uses: actions/checkout@v2
 
-      - name: Select Xcode ${{ matrix.xcode }}
-        run: sudo xcode-select --switch /Applications/Xcode_${{ matrix.xcode }}.app
-
-      - name: Cache SwiftPM
-        uses: actions/cache@v2
-        with:
-          path: CIDependencies/.build
-          key: ${{ runner.os }}-xcode_${{ matrix.xcode }}-swiftpm-ci-deps-${{ github.workspace }}-${{ hashFiles('CIDependencies/Package.resolved') }}
-          restore-keys: |
-            ${{ runner.os }}-xcode_${{ matrix.xcode }}-swiftpm-ci-deps-${{ github.workspace }}
-
-      - name: Cache DerivedData
-        uses: actions/cache@v2
-        with:
-          path: ~/Library/Developer/Xcode/DerivedData
-          key: ${{ runner.os }}-${{ matrix.platform }}_derived_data-xcode_${{ matrix.xcode }}
-          restore-keys: |
-            ${{ runner.os }}-${{ matrix.platform }}_derived_data
-
```

Comment:
Swift doesn't seem to use caches (other than the checkouts) so overall this caching seems to slow things down 😞